diff -r -c3 spl-0.7.1-orig/scripts/kmodtool spl-0.7.1/scripts/kmodtool
*** spl-0.7.1-orig/scripts/kmodtool	2017-08-08 15:58:23.000000000 -0700
--- spl-0.7.1/scripts/kmodtool	2017-08-27 09:51:06.554282611 -0700
***************
*** 153,160 ****
  Provides:         kmod-${kmodname}-uname-r = ${kernel_uname_r}
  Provides:         ${kmodname}-kmod = %{?epoch:%{epoch}:}%{version}-%{release}
  Requires:         ${kmodname}-kmod-common >= %{?epoch:%{epoch}:}%{version}
! Requires(post):   ${prefix}/sbin/depmod
! Requires(postun): ${prefix}/sbin/depmod
  EOF
  
  	if [[ ${obsolete_name} ]]; then
--- 153,161 ----
  Provides:         kmod-${kmodname}-uname-r = ${kernel_uname_r}
  Provides:         ${kmodname}-kmod = %{?epoch:%{epoch}:}%{version}-%{release}
  Requires:         ${kmodname}-kmod-common >= %{?epoch:%{epoch}:}%{version}
! # Requires(post):   ${prefix}/sbin/depmod
! # Requires(postun): ${prefix}/sbin/depmod
! AutoReq:	no
  EOF
  
  	if [[ ${obsolete_name} ]]; then
